client/src/js/field/googlemapsfield.js

```diff
@@ -25,11 +25,13 @@ const fillAddress = () => {
 };
 
 const mountMapsField = (mapsfield) => {
-  options = Object.assign(options, window.customisations);
-  autoComplete = new google.maps.places.Autocomplete(mapsfield, options);
-  autoComplete.addListener('place_changed', fillAddress);
+  const fieldName = mapsfield.name;
+  options = Object.assign(options, window[fieldName + 'Customisations']);
+  autoComplete[fieldName] = new google.maps.places.Autocomplete(mapsfield, options);
```

Comment:
You're treating autoComplete as an array/object, while at the same time treating the fieldName as a unique identifier. This is counter intuitive and bad practice. It's too fragile.